src/Symfony/Component/Cache/Tests/Traits/RedisTraitTest.php

```diff
@@ -143,7 +143,7 @@ public function testDbIndexDsnParameter(string $dsn, int $expectedDb)
             self::markTestSkipped('REDIS_AUTHENTICATED_HOST env var is not defined.');
         }
 
-        $mock = new class () {
+        $mock = new class  {
```

Comment:
Does the fixer produce two spaces between `class` and `{`?

Reply:
The fixer would actually remove the additional space, but that's through another rule (`class_definition`). For this PR, I ran the fixer with only the `new_with_parentheses` rule, otherwise it would fix 94 other files and clutter the diff of this PR.

```
php-cs-fixer fix --rules='{"new_with_parentheses": {"anonymous_class": false}}' 
```

Here's the full diff for this file when running all the rules:

```
src/Symfony/Component/Cache/Tests/Traits/RedisTraitTest.php (new_with_parentheses, class_definition, trailing_comma_in_multiline)
```
 
<img width="745" alt="image" src="https://github.com/user-attachments/assets/32bd9bc1-f44f-4a4e-b1a4-2e08fbe55cee">